src/commands/mod.rs

```diff
@@ -59,6 +62,12 @@ pub enum Command {
     /// Signs a ManageNeuron message to submit a UpgradeSnsControlledCanister
     /// proposal.
     MakeUpgradeCanisterProposal(make_upgrade_canister_proposal::MakeUpgradeCanisterProposalOpts),
+    /// Prints a summary with general info about the sns.
+    SnsSummary(sns_summary::SnsSummaryOps),
+    /// Prints info about the canisters belonging to this sns.
+    SnsCanistersSummary(sns_canisters_summary::SnsCanistersSummaryOps),
+    /// Prints info about the system parameters in this sns.
+    NervousSystemParameters,
```

Comment:
```suggestion
    GetNervousSystemParameters,
```
Please change this and all the subsequent files/structs related to this command